src/saving/SaveListItem.cs

```diff
@@ -77,6 +77,9 @@ public class SaveListItem : PanelContainer
     [Signal]
     public delegate void OnSelectedChanged();
 
+    [Signal]
+    public delegate void OnDoubleClicked(SaveListItem item);
```

Comment:
Hang on why is the signal returning `this`? I'd much prefer so that there isn't a useless parameter (an object sending a signal and saying "hey, this is me" is unnecessary.